delfin/drivers/huawei/oceanstor/rest_client.py

```diff
@@ -49,6 +49,8 @@ def __init__(self, **kwargs):
 
     def init_http_head(self):
         self.url = None
+        if self.session:
```

Comment:
Please verify that this is not called by multiple instances of a process. Else, if one caller is using the session, then others when it comes, may close the session for others.
Will making it singleton help?